CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.15.2 / 2025-05-27
+
+* [CHANGE] Update dependencies
+
+This addresses CVE-2025-22872
```

Comment:
This CVE is related to `golang.org/x/net/html`, which is unused by the exporter code.

We do not list CVEs that are not related to our projects.

```suggestion
```

Reply:
Okay, i only referenced it because security scanners are not smart enough to know

Reply:
Yea, I understand that. But we are trying to not encourage bad security scanner behavior. It's just a lot of toil.